### ISSUE TITLE
front: limit the horizontal scrolling

### ISSUE THREAD

**Issue by clarani** [OpenRailAssociation/osrd#8173]
### Description and goal

The time displayed in the manchette is persistent and never re-centers. I scrolled too far to see if there was a limit, and I actually struggled to go back to the train.

We should limit the horizontal scroll window. Either we restrict it to the days when services are running, or even to the hours when services are running.
=> to discuss with PO

### Acceptance criteria

- [ ] the user can not scrolled too far

**Comment by eckter** [comment]
Scrolling too far isn't necessarily the problem, at some point we'll have timetable that span over a pretty long time period anyway. A different fix would be a way to center the manchette on the selected train. But that would take longer to implement.

**Comment by clarani** [comment]
You're right, maybe we have 2 problems, but the most important one is to be able to center the manchette on the selected train. I create a new ticket for this :+1: https://github.com/OpenRailAssociation/osrd/issues/8177
